modules/http/server/src/test/scala/GreeterRestClients.scala

```diff
@@ -17,6 +17,7 @@
 package freestyle.rpc.http
 
 import cats.effect._
+import freestyle.rpc.http.Utils._
 import fs2.Stream
 import io.circe.generic.auto._
```

Comment:
Possibly, but since the clients calling these methods provide and receive the domain objects I wanted to hide the JSON conversion from them as an implementation detail; I didn't see an advantage in putting the clients in charge of defining and passing the encoders.